### PR TITLE
ability to remove tracks

### PR DESCRIPTION
**Pull request by ShamsGolap**
Finally!

### CODE REVIEW

File: activmusic/apps/uploadmgr/templates/uploadmgr/index.html [hunk 1]
```diff
@@ -8,24 +8,28 @@
...
-    {% endif %}
-    {% empty %}
-        <p>{% trans "You don't have uploaded anything yet!" %}</p>
-    {% endfor %}
+    <form method="post" action="{% url 'music_delete' %}">
+        {% csrf_token %}
+        <button class="btn btn-danger">Remove all selected tracks</button>
+        {% for media in medias %}
+        {% if forloop.first %}
+        <table class="table">
```
Comment (VincentLoy) [inline activmusic/apps/uploadmgr/templates/uploadmgr/index.html]: Why did you not start the form here and put the closing form tag below the {{ if forloop.last }} ? I mean, choosing this way, the form will not appear for people that not have uploaded music yet :)

File: activmusic/apps/uploadmgr/templates/uploadmgr/index.html [hunk 2]
```diff
@@ -8,24 +8,28 @@
...
-    </table>
-    {% endif %}
-    {% empty %}
-        <p>{% trans "You don't have uploaded anything yet!" %}</p>
-    {% endfor %}
+    <form method="post" action="{% url 'music_delete' %}">
+        {% csrf_token %}
+        <button class="btn btn-danger">Remove all selected tracks</button>
+        {% for media in medias %}
+        {% if forloop.first %}
```
Comment (VincentLoy) [inline activmusic/apps/uploadmgr/templates/uploadmgr/index.html]: Why did you not start the form here and put the closing form tag below the {{ if forloop.last }} ? I mean, choosing this way, the form will not appear for people that not have uploaded music yet :)

File: activmusic/apps/uploadmgr/templates/uploadmgr/index.html [hunk 3]
```diff
@@ -10,20 +10,24 @@
...
-        <tr>
-            <td>{{ media.name|default:"[not downloaded yet]" }}</td>
-            <td><a href="{{ media.url }}" target="_blank">{{ media.url }}</a></td>
-            <td><input type="checkbox"/></td>
-        </tr>
-    {% if forloop.last %}
-    </table>
+    <form method="post">
+        {% csrf_token %}
+        <button class="btn btn-danger">Remove all selected tracks</button>
```
Comment (Xowap) [inline activmusic/apps/uploadmgr/templates/uploadmgr/index.html]: That is not translated

File: activmusic/apps/uploadmgr/templates/uploadmgr/index.html [hunk 4]
```diff
@@ -10,20 +10,24 @@
...
+            <tr>
+                <th>{% trans "Name" %}</th>
+                <th>{% trans "URL" %}</th>
+                <th class="table-col-delete">{% trans "Delete" %}</th>
+            </tr>
+        {% endif %}
+            <tr>
+                <td>{{ media.name|default:"[not downloaded yet]" }}</td>
+                <td><a href="{{ media.url }}" target="_blank">{{ media.url }}</a></td>
+                <td><input name="toDelete" value="{{ media.id }}" type="checkbox"/></td>
```
Comment (Xowap) [inline activmusic/apps/uploadmgr/templates/uploadmgr/index.html]: You shall not use camelCase as form field names

File: activmusic/apps/uploadmgr/templates/uploadmgr/index.html (hunk 3 above)
Comment (Xowap) [inline activmusic/apps/uploadmgr/templates/uploadmgr/index.html]: Maybe red is too visible too. Try something more discrete. "Delete" is not an action you want your user to do.

**Pull request merged**